spec.yaml

```diff
@@ -1381,6 +1381,78 @@ paths:
           schema:
             $ref: "./definitions.yaml#/definitions/V4GenericResponse"
 
+  /v4/clusters/{cluster_id}/apps/{app_name}/config/:
+    post:
+      operationId: createClusterAppConfig
+      consumes:
+        - multipart/form-data
+      tags:
+        - apps
+      summary: Create app config
+      description: |
+        This operation allows you to create a values configmap for a specific app. The app does
+        have to exist before hand.
```

Comment:
It sounds weird you are creating a subresource of resource that does not exist. Why do we want to do it like that?
Can be a configmap use for multiple appCRs (same type)?

Reply:
> Can be a configmap use for multiple appCRs

Via the kubernetes API (App CR spec) yes, via the GS API, no.

The App CR spec has an explicit reference to the configmap used for the user values, so anything can go there.

For the case of the API, it's about setting up a convention so that we can get a configmap in place before making the App CR, so that the App gets installed properly in one go.